CMakeLists.txt

```diff
@@ -206,7 +208,13 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
 
 else()
     # Custom build flags.
-    set(QUIC_COMMON_FLAGS "-DQUIC_PLATFORM_LINUX -fms-extensions -fPIC -pthread -Wl,--no-as-needed -ldl")
+
+    set(QUIC_COMMON_FLAGS "-fms-extensions -fPIC -pthread -fno-inline -fno-omit-frame-pointer")
```

Comment:
@Maximus- Were -fno-inline and -fno-omit-frame-pointer required for macOS? If so those should only be added to mac. We don't want these flags unconditionally on linux.

Reply:
Not at all; only there for debugging. Will remove. My mistake!

Reply:
Got the PR in #554 already, so we're good. Just wanted to know if it was something mac required so I could add it to mac specifically.